pandas/tests/arrays/sparse/test_array.py

```diff
@@ -1198,13 +1198,42 @@ def test_from_coo(self):
     def test_to_coo(self):
         import scipy.sparse
 
-        ser = pd.Series(
-            [1, 2, 3],
-            index=pd.MultiIndex.from_product([[0], [1, 2, 3]], names=["a", "b"]),
-            dtype="Sparse[int]",
+        s = pd.Series([np.nan] * 6)
```

Comment:
can you parameterize over sort_lables rather than repeat

Reply:
done